packages/terra-form-field/docs/README.md

```diff
@@ -0,0 +1,35 @@
+# Terra Form Field
+
+The Form Field component by handles the layout of the label, help text and error text associated with a form control Additionally, it provides required, optional, and invalid indicators.
```

Comment:
I think it would be better to say form element. control could potentially have confusion with our control elements of radio and checkbox.

Reply:
Good call. I struggled with the term to use and chose control based on the zep naming. Also I missed the `.` between my sentences. 🙈